app/controllers/spina/registers_controller.rb

```diff
@@ -31,9 +31,9 @@ def show
 
       records =
         case params[:status]
-        when 'closed'
+        when 'archived'
           @register_data.get_expired_records
-        when 'all'
+        when 'both'
```

Comment:
I am not sure `both` makes sense as the param name. As it would be wrong if we added more filters in future. I would rather leave this as `all`